packages/icons/icons.json

```diff
@@ -4373,5 +4373,110 @@
         "tags": "layer, map, reference, point, geography, distance, boundry",
         "group": "miscellaneous",
         "codepoint": 62321
+    },
+    {
+        "displayName": "Block promote",
+        "iconName": "block-promote",
+        "tags": "upgrade, line, chevrons",
```

Comment:
For any icon with the double chevrons, should we consider aligning with the same categories used for the `chevron-right` icon? Or some of them? (e.g. Arrow, direction, next, forward)

If we searched for "chevron" in this case, would it return this icon or does the plural have to be specified?

Reply:
Yeah, we can certainly align these!

And yeah, to answer your question, it would return this icon. We currently do a fairly rudimentary "smart search" to filter these icons, which is really just a substring match: https://github.com/palantir/blueprint/blob/39f0a833ee26fd0626a37c9fc090d6916f0ebba4/packages/docs-theme/src/common/stringUtils.ts#L17-L21

Reply:
Also, the [deploy preview](https://output.circle-artifacts.com/output/job/1761bb7c-db4d-4150-a47c-125c99b5260c/artifacts/0/packages/docs-app/dist/index.html) is available to test out search queries. 🙂